src/App.tsx

```diff
@@ -197,6 +119,73 @@ class App extends React.Component {
       </Router>
     );
   }
+
+  private getCategoryDropdownId(categoryName: string) {
```

Comment:
Are the id's actually used for anything now?  We're not doing much with css.

Reply:
They are required for the control.  I tried to leave them off, but you get a build error.  I didn't bother looking too deeply into why.

Reply:
```
/Users/kfarnung/Git/GitHub/puzztool/src/App.tsx
(130,10): Type '{ children: Element[]; eventKey: string; title: string; }' is not assignable to type 'Readonly<NavDropdownProps>'.
  Property 'id' is missing in type '{ children: Element[]; eventKey: string; title: string; }'.
```